lib/SiftResponse.php

```diff
@@ -12,23 +12,23 @@ public function __construct($result, $httpStatusCode, $request) {
         $this->body = null;
         $this->apiStatus = null;
         $this->apiErrorMessage = null;
-
-        if (function_exists('json_decode')) {
```

Comment:
The result of this code is overridden in the `if` below (except for status 204/304, which should have no response body, anyway)